pkgs/development/haskell-modules/default.nix

```diff
@@ -36,6 +37,8 @@ let
     compilerConfig
     packageSetConfig
     overrides
+  ] ++ lib.optionals stdenv.targetPlatform.isGhcjs [
+    (configurationJS { inherit pkgs haskellLib; })
```

Comment:
I tried to add this before, alongside the other `platformConfigurations`, but the overrides would get lost as `configuration-ghc-9.8.nix`  (passed in `compilerConfig`) would then reset to `null` or overwrite due to picking the most-recent hackage version.

Reply:
That’s unfortunate.

Reply:
I don’t get it. there do not seem to be conflicting overrides for the packages in ghcjs-9.x.nix after the changes you are making. Are you sure this doesn’t work?

Reply:
> Are you sure this doesn’t work?

Confirmed to be working now. I think the problem is that I originally had
```
    aeson = disableLibraryProfiling super.aeson;
```
which was being replaced by the flat `aeson` override in the 9.8 bit but now we disable profiling globally for JS flavors so this is no longer a problem.